samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -44,63 +69,14 @@ private void Update()
         {
             mapComponent.APIKey = APIKey;
         }
+
+        SceneLoadedCount = SceneManager.sceneCount;
     }
 
-    private void Start()
+    // Read string from the input field for the API key
```

Comment:
Ehh, you dont really need this comment, the function name is pretty self explanatory and isn't educational sample code
```suggestion
```